lepidopter-fh/setup-ooniprobe.sh

```diff
@@ -18,8 +18,14 @@ apt-get -y install -t stretch obfs4proxy
 
 # Remove previous system versions of pyasn1 and python-cryptography
 apt-get -y remove python-pyasn1 python-cryptography
-# Install ooniprobe obfsproxy and fteproxy
-pip install ooniprobe obfsproxy fteproxy
+# Install obfsproxy and fteproxy
+pip install obfsproxy fteproxy
+# Install ooniprobe v2.0.0 alpha
+pip install -e \
+    git+https://github.com/TheTorProject/ooni-probe@v2.0.0-alpha#egg=ooniprobe
```

Comment:
Note that the branch on github doesn't include a built version of the web UI, so it will not work.

I have uploaded a package that includes also the web UI here: https://people.torproject.org/~art/ooni/ooniprobe-2.0.0rc1.tar.gz.

You should use this for testing and development.